common/models.py

```diff
@@ -395,19 +395,19 @@ class CustomPage(Page, index.Indexed):
     def serve(self, request):
         return render(request, self.template, {
             'page': self,
-            'people': Person.objects.all(),
+            'people': Person.objects.all().prefetch_related('photo'),
             'team': Person.objects.filter(
                 tags__name__in=['team']
-            ).order_by('last_name'),
+            ).prefetch_related('photo').order_by('last_name'),
             'alumni': Person.objects.filter(
                 tags__name__in=['alum']
-            ).order_by('last_name'),
+            ).prefetch_related('photo').order_by('last_name'),
```

Comment:
Drive-by: since we don't show photos for alumni, there's no need to prefetch it.

In fact, you can use `.only` to load only the fields we need. I think we only need the first and last name fields.